it/repositories/ArrivalMovementRepositorySpec.scala

```diff
@@ -34,7 +34,18 @@ class ArrivalMovementRepositorySpec
     with EitherValues
     with TryValues
     with ModelGenerators
-    with MongoDateTimeFormats {
+    with MongoDateTimeFormats with BeforeAndAfterAll {
+
+  override def afterAll(): Unit = {
+    super.afterAll()
+    database.flatMap {
+      db =>
+        val arrivalMovement1 = arbitrary[Arrival].sample.value
+        val arrivalMovement2 = arbitrary[Arrival].sample.value
+
+        db.collection[JSONCollection](ArrivalMovementRepository.collectionName).insert(false).many(Seq(arrivalMovement1, arrivalMovement2))
+    }.futureValue
+  }
```

Comment:
Why do we need to insert two new documents after the test suite is run? Isnt afterAll mostly used for cleanup purposes?

Reply:
Will remove, was used while debugging